components/adblock_rust_ffi/src/lib.rs

```diff
@@ -343,7 +343,7 @@ pub unsafe extern "C" fn filter_list_metadata_title(
 }
 
 #[no_mangle]
-pub const SUBSCRIPTION_DEFAULT_EXPIRES_HOURS: u16 = 7 * 24;
+pub static SUBSCRIPTION_DEFAULT_EXPIRES_HOURS: u16 = 7 * 24;
```

Comment:
Can we remove the `pub` and `no_mangle` annotation instead? There seems to a duplicate `SUBSCRIPTION_DEFAULT_EXPIRES_HOURS` in lib.h which is what the code actually uses.

Reply:
doesn't it use it right here? I'm happy to remove it or so something so we only have one copy of it. It seems silly to have this be `no_mangle` in the first place if we're not using it outside of the file 
```
#[no_mangle]
pub unsafe extern "C" fn filter_list_metadata_expires(metadata: *const FilterListMetadata) -> u16 {
    use adblock::lists::ExpiresInterval;

    match (*metadata).expires.as_ref() {
        Some(ExpiresInterval::Days(d)) => 24 * *d as u16,
        Some(ExpiresInterval::Hours(h)) => *h,
        None => SUBSCRIPTION_DEFAULT_EXPIRES_HOURS,
    }
}
```

Reply:
Sorry, forgot this was for manually-run-and-checked-in `cbindgen`. This is fine for now. See also https://github.com/brave/brave-core/pull/17368.